cps/kobo.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import base64
+import copy
+import uuid
+import os
+from datetime import datetime, tzinfo, timedelta
+from time import gmtime, strftime
+
+from b2sdk.account_info.in_memory import InMemoryAccountInfo
+from b2sdk.api import B2Api
```

Comment:
To me this looks like remains from the backblaze integration. There is the not called function (get_download_url_for_book_b2) which needs this. From my point of view, this should be be removed including the optional requirement for b2sdk

Reply:
Yep, this is a leftover; removed occurrences from this file, optional-requirements and .gitignore.